airbyte-integrations/connectors/source-mysql-v2/src/test/resources/expected-spec.json

```diff
@@ -1,363 +1,363 @@
 {
```

Comment:
Not sure why the diff is so noisy in this file. Lots of lines moved around. My version was serialized using `Jsons`, perhaps the previous one wasn't. The diff checker doesn't care, fortunately.